src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -211,29 +211,43 @@
     "<$0.01"
     (prettify-balance currency-symbol fiat-value)))
 
+(defn prettify-percentage-change
+  "Returns unsigned precentage"
+  [percentage]
+  (-> (if (number? percentage) percentage 0)
+      money/bignumber
+      money/absolute-value
+      (money/to-fixed 2)))
 
 (defn calculate-token-value
   "This function returns token values in the props of token-value (quo) component"
   [{:keys [token color currency currency-symbol]}]
-  (let [token-units                 (total-token-units-in-all-chains token)
-        fiat-value                  (total-token-fiat-value currency token)
-        market-values               (or (get-in token [:market-values-per-currency currency])
-                                        (get-in token
-                                                [:market-values-per-currency
-                                                 constants/profile-default-currency]))
-        {:keys [change-pct-24hour]} market-values
-        crypto-value                (get-standard-crypto-format token token-units)
-        fiat-value                  (get-standard-fiat-format crypto-value currency-symbol fiat-value)]
+  (let [token-units                       (total-token-units-in-all-chains token)
+        fiat-value                        (total-token-fiat-value currency token)
+        market-values                     (or (get-in token [:market-values-per-currency currency])
+                                              (get-in token
+                                                      [:market-values-per-currency
+                                                       constants/profile-default-currency]))
+        {:keys [price change-pct-24hour]} market-values
+        formatted-token-price             (prettify-balance currency-symbol price)
+        percentage-change                 (prettify-percentage-change change-pct-24hour)
+        crypto-value                      (get-standard-crypto-format token token-units)
+        fiat-value                        (get-standard-fiat-format crypto-value
+                                                                    currency-symbol
+                                                                    fiat-value)]
     {:token               (:symbol token)
      :token-name          (:name token)
      :state               :default
+     :metrics?            true
```

Comment:
not for this pr but is metrics always on? 🤔 - or is there some cases where we don't want it?

Reply:
For now, yes (as we show the token market price).

In future, we will be showing the fiat change where we would hide the metrics for zero balance tokens.